mobile/src/services/AnalyticsService.ts

```diff
@@ -1,4 +1,4 @@
-import * as Sentry from 'sentry-expo';
+// import * as Sentry from 'sentry-expo';
```

Comment:
Can you please leave a comment explaining why these lines are commented out and what needs to be done to uncomment them again?

Reply:
sentry-expo has been causing build errors, this is a separate issue we will have to look into if we want to use sentry. Ive added a comment